network_custom_mtu/main.tf

```diff
@@ -1,6 +1,5 @@
 # [START vpc_auto_create]
 resource "google_compute_network" "vpc_network" {
-  project                 = "my-project-name"
```

Comment:
Should we add:

```
data "google_project" "project" {
}
```

Reply:
Similar to above, we can continue the conversation there.